test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,17 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+  const names1 = [{name: 'Homer'}, {name: 'Marge'}, {name: 'Bart'}, {name: 'Lisa'}, {name: 'Maggie'}];
```

Comment:
When there is an array of objects and it is a bit long, you could reformat it like this:
`````
    const names1 = [
      { name: "Homer" },
      { name: "Marge" },
      { name: "Bart" },
      { name: "Lisa" },
      { name: "Maggie" },
    ];
`````
To make it look a little bit tidier, and easier to read.
Also at some place you use '' and at some place you are using "". For better consistency I would recommend to use "" everywhere.